src/test/java/core/basesyntax/dao/StorageDaoImplTest.java

```diff
@@ -0,0 +1,122 @@
+package core.basesyntax.dao;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class StorageDaoImplTest {
+    private static final int LOWEST_VALID_AGE = 18;
+    private static final int ADULT_AGE = 32;
+    private static final String LOWEST_VALID_PASS = "qw21re";
+    private static User validUser1;
+    private static User validUser2;
+    private static User validUser3;
+    private StorageDao storageDao;
+    private int expectingSize;
+
+    @BeforeAll
+    static void beforeAll() {
+        validUser1 = new User();
+        validUser2 = new User();
+        validUser3 = new User();
+    }
+
+    @BeforeEach
+    void setUp() {
+        storageDao = new StorageDaoImpl();
+
+        validUser1.setId(101L);
+        validUser1.setLogin("ValidUser1");
+        validUser1.setPassword("q1w2e3r4");
+        validUser1.setAge(ADULT_AGE);
```

Comment:
let's create a separate method that will return a user, so we could avoid duplication, in this method you should pass all needed values (id, login, password, age)
the same for the next class